projects/dotcms-field-elements/src/components/dot-textarea/dot-textarea.tsx

```diff
@@ -55,7 +56,7 @@ export class DotTextareaComponent {
     validationMessage = 'The field doesn\'t comply with the specified format';
 
     /** (optional) Disables field's interaction */
-    @Prop({ reflectToAttr: true })
+    @Prop({ mutable: true, reflectToAttr: true })
```

Comment:
We need to talk if we get rid of this prop in favor of `dotxxx` format to pass attr.